tests/unit/adapters/osf-adapter-test.ts

```diff
@@ -47,11 +47,9 @@ module('Unit | Adapter | osf-adapter', hooks => {
     test('#buildURL uses relationship links if available for delete, update, and find', function(assert) {
         const url = 'http://localhost:8000/v2/users/me/rel/';
         const adapter = this.owner.lookup('adapter:osf-adapter');
-        const user = FactoryGuy.make('user', {
-            links: {
-                self: url,
-            },
-        });
+        const user = FactoryGuy.make('user');
+        user.set('links', { self: url });
+
```

Comment:
[ember-data-factory-guy 3.2.1](https://github.com/danielspaniel/ember-data-factory-guy/releases/tag/v3.2.1) makes some changes breaking changes to links.